src/datapump/sync/sync.py

```diff
@@ -723,6 +723,125 @@ def get_latest_release(self) -> Tuple[str, List[str]]:
+        source_uris = [
+            f"gs://{self.source_bucket}/{self.source_prefix}/VEG-ANOM-MAX",
+            f"gs://{self.source_bucket}/{self.source_prefix}/VEG-DIST-DATE",
+            f"gs://{self.source_bucket}/{self.source_prefix}/VEG-DIST-COUNT",
+        ]
+
+        # This file is updated once tiles are updated
+        upload_date_text = get_gs_file_as_text(
+            self.source_bucket, f"{self.source_prefix}/uploadDate.txt"
+        )
+
+        # Example string: "Updated Fri Apr 15 14:27:01 2022 UTC"
+        upload_date = upload_date_text[12:-5]
+        LOGGER.info(f"Last DIST-Alert upload date: {upload_date}")
+        latest_release = datetime.strptime(upload_date, "%b %d %H:%M:%S %Y").strftime(
+            "v%Y%m%d"
+        )
+
+        return latest_release, source_uris
+    
+    def build_jobs (self, config: DatapumpConfig) -> List[Job]:
+        latest_api_version = self.get_latest_api_version(self.dataset_name)
+        latest_release, source_uris = self.get_latest_release()
+
+        # If the latest API version matches latest release from UMD, no need to update
+        if latest_api_version == latest_release:
+            return []
+        
+        jobs: List[Job] = []
+
+        job = RasterVersionUpdateJob(
+            # Current week alerts tile set
+            id=str(uuid1()),
+            status=JobStatus.starting,
+            dataset=self.dataset,
+            version=latest_release,
+            tile_set_parameters=RasterTileSetParameters(
+                source_uri=source_uris,
+                calc=self.input_calc,
+                grid="10/40000",
+                data_type="int16",
+                no_data=-1,
+                pixel_meaning="currentweek",
+                band_count=1,
+                compute_stats=False,
+                union_bands=True,
+                unify_projection=True
+            ),
+            content_date_range=ContentDateRange(
+                start_date="2020-12-31", end_date=str(date.today())
+            )
+        )
+        job.aggregated_tile_set_parameters = AuxTileSetParameters(
+            # Aggregated tile set (to include all alerts)
+            pixel_meaning="default",
+            grid="10/40000",
+            data_type="int16",
+            no_data=-1,
+            calc="np.where(A > 0, A, B)",
+            auxiliary_asset_pixel_meaning = "default"
+        )
+        job.aux_tile_set_parameters = [
+            # Intensity tile set
+            AuxTileSetParameters(
+                source_uri=None,
+                pixel_meaning="intensity",
+                data_type="uint8",
+                calc="(B > 0) * 55",
+                grid="10/40000",
+                no_data=None,
+                auxiliary_asset_pixel_meaning = "default"
+            )
+        ]
+        job.cog_asset_parameters = [
+            # Created from the "default" asset
+            CogAssetParameters(
+                source_pixel_meaning="date_conf",
```

Comment:
This should be 'default', since we want to use the aggregated asset (which has pixel_meaning 'default').  This matches well, since we're creating a COG implementation called 'default'. too.  'date_conf' was from IntegratedAlerts.